README.md

```diff
@@ -73,9 +73,16 @@ Open your browser to the URL shown.
 To access the database from the commannd line:
 
 ```bash
-psql -p 5432 -h localhost -U postgres -d rust_contributors
+psql -p 5432 -h localhost -U postgres -d rust_thanks
```

Comment:
what about maybe just 'thanks' here instead of `rust_thanks`? I named it with the `rust_` prefix mostly by accident.